sql/mob_pools.sql

```diff
@@ -1697,7 +1697,7 @@ INSERT INTO `mob_pools` VALUES (1641,'Goblin_Bouncer','Goblin_Bouncer',133,0x000
 INSERT INTO `mob_pools` VALUES (1642,'Goblin_Bounty_Hunter','Goblin_Bounty_Hunter',133,0x0000F10100000000000000000000000000000000,1,1,3,240,100,0,1,0,1,0,0,0,0,131,0,0,0,0,0,133,133);
 INSERT INTO `mob_pools` VALUES (1643,'Goblin_Butcher','Goblin_Butcher',133,0x0000EF0100000000000000000000000000000000,1,1,4,240,100,0,1,0,1,0,0,0,8067,131,0,0,0,0,0,133,133);
 INSERT INTO `mob_pools` VALUES (1644,'Goblin_Chapman','Goblin_Chapman',133,0x0000B30200000000000000000000000000000000,4,4,12,240,100,0,1,0,1,0,0,0,835,131,0,0,2,0,0,133,133);
-INSERT INTO `mob_pools` VALUES (1645,'Goblin_Collector','Goblin_Collector',133,0x0000BC0200000000000000000000000000000000,3,3,11,220,100,0,1,0,1,2,0,0,0,3,0,0,1,0,0,133,133);
+INSERT INTO `mob_pools` VALUES (1645,'Goblin_Collector','Goblin_Collector',133,0x0000BC0200000000000000000000000000000000,3,3,11,150,200,0,1,0,1,2,17,0,0,3,0,0,463,0,0,1166,133);
```

Comment:
The "fast attacks" mentioned on wiki appear to be bogus. I just checked him seemed to be the normal 240 most mobs get except that he has a high rate of triple attack (more consistent with a thf main, which makes sense when you realize mobs don't really do the half-level sub thing the way players do they get full traits of both jobs).

when an entity attacks the combat round timer counts from the begging, so you can start your next attack as the 3rd hit end. My thf on retail can look like hundred fists with some haste and my triple attacks going off.

fortunately, I don't need to record this because somebody already did so you can see the 3 attacks with a very brief pause: https://youtu.be/muuYcCtqmXo?t=27

Reply:
I've corrected his job to THF and removed the lowered delay. Also realized there is a line that is said when spawning him so added that as well. Added a TODO to eventually find the jumping animation he does on spawn.

Reply:
I don't know that he is a thf main - just that he has that much triple attack. I don't know if it being his main or his sub even matters, but given that both wiki's say when there is a high probability someone in the future will see that and come back to "fix" his job here, even if  turns out its right (we just don't know yet).

Given that, I think we shoudl play it safe and use `3,6` than to `6,3` here for the time being.